pyproject.toml

```
@@ -27,9 +27,9 @@ license = {file = "LICENSE"}
 dependencies = [
     "pydantic ==2.*",
     "polars ~= 1.0",
-    "ga4gh.vrs ==2.0.0a10",
+    "ga4gh.vrs ==2.*",
     "biocommons.seqrepo",
-    "gene-normalizer ==0.4.0",
+    "gene-normalizer ==0.9.0",
```

I'd recommend not pinning the exact version. 

```suggestion
    "gene-normalizer ~=0.9.0",
```